core/kt-osrd-utils/src/main/kotlin/fr/sncf/osrd/utils/indexing/StaticIdx.kt

```diff
@@ -188,3 +188,8 @@ class VirtualStaticPool<IndexT>(private var _size: UInt) : StaticIdxIterable<Ind
         return StaticIdxSpace(size)
     }
 }
+
+/** Utility function to instantiate ids from java code, which can't really handle UInts */
```

Comment:
Oh, true.

My reasoning was: I apparently can't directly instantiate a value type from java, so I tried wrapping the constructor in a function. The Int/UInt thing wasn't actually a type error as I though, but rather a mangling problem, adding `@JVMName` would have fixed it.

But you're right, there's actually no need for a function in the first place: because in java the conversion from int -> UInt -> StaticIdx doesn't actually do anything, I can just do `loadingGauge.isCompatibleWith(gaugeType.ordinal())